code/modules/hydroponics/grown/melon.dm

```diff
@@ -12,7 +12,7 @@
 	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
 	icon_dead = "watermelon-dead"
 	genes = list(/datum/plant_gene/trait/repeated_harvest)
-	mutatelist = list(/obj/item/seeds/watermelon/holy)
+	mutatelist = list(/obj/item/seeds/watermelon/holy, /obj/item/seeds/superwatermelon)
```

Comment:
you forgot to do what you said that you'd do, and watermelons are still able to mutate directly into holymelons